web/src/lib/components/faces-page/person-side-panel.svelte

```diff
@@ -207,91 +204,125 @@
         </div>
       {:else}
         {#each peopleWithFaces as face, index}
-          {#if face.person}
-            <div class="relative z-[20001] h-[115px] w-[95px]">
-              <div
-                role="button"
-                tabindex={index}
-                class="absolute left-0 top-0 h-[90px] w-[90px] cursor-default"
-                on:focus={() => ($boundingBoxesArray = [peopleWithFaces[index]])}
-                on:mouseover={() => ($boundingBoxesArray = [peopleWithFaces[index]])}
-                on:mouseleave={() => ($boundingBoxesArray = [])}
-              >
-                <div class="relative">
-                  {#if selectedPersonToCreate[face.id]}
-                    <ImageThumbnail
-                      curve
-                      shadow
-                      url={selectedPersonToCreate[face.id]}
-                      altText={selectedPersonToCreate[face.id]}
-                      title={'New person'}
-                      widthStyle={thumbnailWidth}
-                      heightStyle={thumbnailWidth}
-                    />
-                  {:else if selectedPersonToReassign[face.id]}
+          {@const person_name = face.person ? face.person?.name : 'unassigned'}
+          <div class="relative z-[20001] h-[115px] w-[95px]">
+            <div
+              role="button"
+              tabindex={index}
+              class="absolute left-0 top-0 h-[90px] w-[90px] cursor-default"
+              on:focus={() => ($boundingBoxesArray = [peopleWithFaces[index]])}
+              on:mouseover={() => ($boundingBoxesArray = [peopleWithFaces[index]])}
+              on:mouseleave={() => ($boundingBoxesArray = [])}
+            >
+              <div class="relative">
+                {#if selectedPersonToCreate[face.id]}
+                  <ImageThumbnail
+                    curve
+                    shadow
+                    url={selectedPersonToCreate[face.id]}
+                    altText={selectedPersonToCreate[face.id]}
+                    title={'New person'}
+                    widthStyle={thumbnailWidth}
+                    heightStyle={thumbnailWidth}
+                  />
+                {:else if selectedPersonToReassign[face.id]}
+                  <ImageThumbnail
+                    curve
+                    shadow
+                    url={getPeopleThumbnailUrl(selectedPersonToReassign[face.id].id)}
+                    altText={selectedPersonToReassign[face.id]?.name || selectedPersonToReassign[face.id].id}
```

Comment:
I know this is existing code, but I don't see the point of having an ID for alt text. It's arguably worse than not having anything there because it will make a screen reader speak gibberish.

Reply:
I agree with that and removed the ID from every altText you mentioned.